pontoon/base/models/changed_entity_locale.py

```diff
@@ -1,15 +1,20 @@
+from typing import TYPE_CHECKING
+
 from django.db import models
 from django.utils import timezone
 
+if TYPE_CHECKING:
+    from pontoon.base.models import Entity, Locale
+
 
 class ChangedEntityLocale(models.Model):
     """
     ManyToMany model for storing what locales have changed translations for a
     specific entity since the last sync.
     """
 
-    entity = models.ForeignKey("Entity", models.CASCADE)
-    locale = models.ForeignKey("Locale", models.CASCADE)
+    entity: models.ForeignKey["Entity"] = models.ForeignKey("Entity", models.CASCADE)
+    locale: models.ForeignKey["Locale"] = models.ForeignKey("Locale", models.CASCADE)
```

Comment:
Because elsewhere the ForeignKey constraints are using the actual values and here they're strings, to avoid circular dependencies; the type resolution isn't able to follow that to figure out what these are foreign keys of.